moped-editor/src/views/dashboard/DashboardView.js

```diff
@@ -95,8 +95,8 @@ const TABS = [
 
 const DashboardView = () => {
   const userSessionData = getSessionDatabaseData();
-  const userId = userSessionData.user_id;
-  const userName = userSessionData.first_name;
+  const userId = userSessionData?.user_id;
+  const userName = userSessionData?.first_name;
```

Comment:
Just doing this because when Rebecca was signing into test, her `userSessionData` was undefined, so this was throwing an error. This won't fix her problem, but it won't WSOD now.